app/app.go

```diff
@@ -1849,6 +1945,22 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 			panic(err)
 		}
 	}
+
+	//if app.mevConfig.ListenAddr != "" {
+	//	err := mevbase.StartServer(app.Logger(), app.mevConfig, app.MevKeeper, ctxProvider)
+	//	if err != nil {
+	//		panic(err)
+	//	}
+	//}
+
+	fmt.Printf("MEV RPC Poller address %s\n", app.mevConfig.ServerAddr)
+	if app.mevConfig.ServerAddr != "" {
+		_, err := mevbase.NewPoller(app.Logger(), app.mevConfig, app.MevKeeper, func() int64 { return app.LastBlockHeight() })
```

Comment:
you could just pass `app.LastBlockHeight` rather than wrapping it in a function

Reply:
> you could just pass `app.LastBlockHeight` rather than wrapping it in a function

Yes, great spot, such a silly mistake.

> but we (ideally) need this to be able to take in a list of serveraddresses (all the bundle servers) that way ultimately the community can decide as many as they want to include.

I don't think that a good idea, as it opens an array of problems - ie how to handle conflicting bundles, or which are more important. If a validator wants to handle more then one bundle-server I think it should rather have, say, aggregating bundle-server which does that and provide single stream for a validator.
Unless I misunderstood the comment and we want to have a fallback option, so you can specify multiple servers in case one goes down.